modules/pool/subgraph-mapper.ts

```diff
@@ -193,5 +193,8 @@ export type FxData = ReturnType<typeof fx>;
 export type GyroData = ReturnType<typeof gyro>;
 export type ElementData = ReturnType<typeof element>;
 export type StableData = ReturnType<typeof stable>;
-export type QuantAmmWeightedData = ReturnType<typeof quantAmmWeighted>;
+export type QuantAmmWeightedData = ReturnType<typeof quantAmmWeighted> & {
+    firstFourWeightsAndMultipliers?: string[];
+    secondFourWeightsAndMultipliers?: string[];
+};
```

Comment:
Why did you have to add them "manually" here? Why isn't that already picked up by the return type?
I think it works for me as is - I'm just wondering why it's different from other pool types.